packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -142,58 +219,96 @@ function VisXZoom({
+              move={move}
+              transformMatrix={_zoom.transformMatrix}
+              transform={_zoom.toString()}
+            >
+              <ZoomEventLayer
+                focusable
+                height={height}
+                onDoubleClick={onDoubleClick}
+                onKeyDown={onKeyZoom}
+                onPointerEnter={onPointerEnter}
+                onPointerDown={panning ? _zoom.dragStart : DEFAULT_HANDLER}
+                onPointerMove={panning ? _zoom.dragMove : DEFAULT_HANDLER}
+                onPointerUp={panning ? _zoom.dragEnd : DEFAULT_HANDLER}
+                onPointerLeave={onPointerLeave}
+                onWheel={onWheel}
+                panning={panning}
+                tabIndex={0}
+                width={width}
+              />
+            </ZoomingComponent>
+          </g>
         )
       }}
     </Zoom>
   )
 }
 
 VisXZoom.propTypes = {
+  /** Allow window scrolling with the mouse wheel. */
+  allowsScrolling: PropTypes.bool,
+  /** 
+   * Custom constraints on the SVG transformation matrix.
+   * https://airbnb.io/visx/docs/zoom#Zoom_constrain
+  */
   constrain: PropTypes.func,
+  /** Height in SVG viewport. */
   height: PropTypes.number.isRequired,
+  /** Left coordinate in SVG viewport. */
   left: PropTypes.number,
+  /** True if pan and zoom is enabled in the subject toolbar. */
+  move: PropTypes.bool,
+  /** Enable panning. Ignored if zooming is false. */
   panning: PropTypes.bool,
+  /** Set the subject toolbar's onPan callback in the classifier store. */
   setOnPan: PropTypes.func,
+  /** Set the subject toolbar's onZoom callback in the classifier store. */
   setOnZoom: PropTypes.func,
+  /** Top coordinate in SVG viewport. */
   top: PropTypes.number,
+  /** Width in SVG viewport. */
   width: PropTypes.number.isRequired,
+  /** Zoom configuration for the subject or workflow. */
   zoomConfiguration: PropTypes.shape({
+    /** Allowed directions for zoom. */
     direction: PropTypes.oneOf(['both', 'x', 'y']),
+    /** Minimum zoom scale. */
     minZoom: PropTypes.number,
+    /** Maximum zoom scale. */
     maxZoom: PropTypes.number,
+    /** Zoom in scale step. */
     zoomInValue: PropTypes.number,
+    /** Zoom out scale step. */
     zoomOutValue: PropTypes.number
   }),
+  /** Enable zooming. true by default. */
   zooming: PropTypes.bool,
+  /** A React component to zoom. It must render SVG, **not** HTML.
+   * `VisXZoom` will inject the following props into this component:
+   * - `initialTransformMatrix`: the initial transformation matrix.
+   * - `transformMatrix`: the current transformation matrix.
+   * - `transform`: a string representation of the matrix transform.
+   * - `children`: `ZoomEventLayer`, an SVG `<rect>` which handles zoom pointer and wheel events.
+   * ```jsx
+   * ({ children, ...zoomProps }) => (
+   *   <SVGComponent {...zoomProps} {...SVGComponentProps} >
+   *     {children}
+   *   </SVGComponent>
```

Comment:
The design of `VisXZoom` means you can't use it to zoom an SVG element that doesn't accept children.

For example, you can't directly zoom an SVG image. The following code will render an image but no event layer to pand and zoom: 
```jsx
<VisXZoom
  zoomingComponent={(props) => <image {...props} height={500} width={1000} src={someURL} />}
/>
```